fetch/main.js

```diff
@@ -0,0 +1,13 @@
+const fetchResponse = fetch('https://jsonplaceholder.typicode.com/users', {
+  method: 'GET'
+})
+  .then(response => response.json(fetchResponse))
+  .catch(console.error);
+console.log(fetchResponse);
```

Comment:
Although it looks like this is working it is not. You need a second `.then` to get your data. Should look like:

```JAVASCRIPT
fetch('https://jsonplaceholder.typicode.com/users', {
  method: 'GET'
})
  .then(response => response.json(fetchResponse))
  .then(data => console.log(data))
  .catch(console.error);
```